readme.md

```diff
@@ -136,6 +136,17 @@ Client-side JS code coverage using [PhantomJS](https://github.com/ariya/phantomj
 lab -r lcov && codecov
 ```
 
+### [nyc](https://github.com/bcoe/nyc)
+```javascript
+{
+  "scripts": {
+    "report-coverage": "nyc report --reporter=lcov | codecov",
```

Comment:
This is incorrect. It should probably be

```sh
nyc report -r lcovonly && codecov
```

Piping the `nyc` output there doesn't do anything. Additionally, for just reporting, `lcovonly` is the only required report.